packages/core/test/lib/utils/traceData.test.ts

```diff
@@ -62,22 +54,19 @@ describe('getTracingMetaTags', () => {
     );
 
     expect(tags).toEqual({
```

Comment:
l: due to above's suggestion

```suggestion
    expect(traceData).toEqual({
```